microk8s-resources/actions/dashboard-ingress.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     nginx.ingress.kubernetes.io/whitelist-source-range: "127.0.0.1/8,169.254.0.0/16,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16"
 
     # redirect and passthrough to dashboard SSL
-    nginx.ingress.kubernetes.io/ssl-redirect: "true"
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
```

Comment:
Redirect did not occur without `force-ssl-redirect`. This redirect is necessary as Kubernetes Dashboard forces HTTPS for logins.